egs/babel_multilang/s5/local/nnet3/run_ivector_common_langs.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# This scripts generates iVector using global iVector extractor
```

Comment:
usually run_ivector_common*.sh also train the ivector extractor. Please rename this script.